include/curl/curl.h

```diff
@@ -165,6 +165,13 @@ typedef enum {
 #define CURLSSLBACKEND_CYASSL CURLSSLBACKEND_WOLFSSL
 #define CURLSSLBACKEND_DARWINSSL CURLSSLBACKEND_SECURETRANSPORT
 
+/* bits for the CURLOPT_FOLLOWLOCATION option */
+#define CURLFOLLOW_ENABLE (1<<0) /* generic follow redirects */
+
+/* Follow redirects, and don't use the custom method for anything but the
+   initial request if the response code says so. */
+#define CURLFOLLOW_IGNORE_CUSTOM (1<<1)
```

Comment:
What do you think about CURLFOLLOW_IGNORE_CUSTOMREQUEST, it's a mouthful but maybe easier for a reader to understand that it's related to CURLOPT_CUSTOMREQUEST. CURLFOLLOW_NO_CUSTOMREQUEST? Probably ambiguous because it could be interpreted as don't follow on customrequest. Actually they both could be interpreted that way. Not particularly committed to any of it, I just don't know if I would readily pick up on what IGNORE_CUSTOM means if I saw it in code. Also do you have any plans to expose this in the tool? --redirect-ignore-customrequest or something?

Reply:
Naming is hard! But yes, I probably need to do something like you suggest to make it slightly more obvious to the reader.

I'm sure this will/should end up in the tool too but I'm not sure under what name/how. I prefer shorter names there but this is a hard one to make short...